app/helpers/format_content_helper.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module FormatContentHelper
+  def auto_link_and_simple_format_content(content:, classname: nil)
+    allowed_attributes = Rails::Html::Sanitizer.white_list_sanitizer.allowed_attributes + %w[href target]
+
+    auto_link_content = auto_link(content.to_s, html: { target: "_blank" }, sanitize: false)
```

Comment:
What happens if you have `sanitize: true` here?

Reply:
It will remove the href and target attributes that we want to keep

Reply:
That seems like a self-defeating option for an `#auto_link` method 🤣

Reply:
It's so we can enable users to enter some html instead of just a string with a link

Reply:
ohhh, yes, got it!

Reply:
I think if you take it out, this [spec](https://github.com/unboxed/bops/pull/883/files#diff-73cf68088b043c75735045653397a26e21544d48d508d0da92abc26b31dcb77fR21-R27) would fail